pkg/controller/user/user_controller.go

```diff
@@ -483,6 +489,59 @@ func (c *Controller) ldapSync(user *iamv1alpha2.User) error {
 	}
 }
 
+func (c *Controller) deleteRoleBindings(user *iamv1alpha2.User) error {
+	listOptions := metav1.ListOptions{LabelSelector: labels.SelectorFromSet(labels.Set{iamv1alpha2.UserReferenceLabel: user.Name}).String()}
+	if result, err := c.ksClient.IamV1alpha2().GlobalRoleBindings().List(listOptions); err != nil {
```

Comment:
No need to check the existence first, just issue the delete request, ignore the NotFoud error.

Reply:
The logic like `kubectl delete <resources> -l  <key>=<value>`.